src/App.js

```diff
@@ -1,12 +1,24 @@
-/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable import/extensions */
```

Comment:
- [ ] I noticed that you disable **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1:

Reply:
thanks for the note @codecaiine